packages/terra-core-docs/src/terra-dev-site/doc/action-header/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,233 @@
+import { Badge } from 'terra-action-header/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import ActionHeaderExample from "./example/BackActionHeader"
+import BackActionHeader from "./example/BackPreviousNextCloseActionHeader"
+import MaximizeActionHeader from "./example/MaximizeCloseActionHeader"
+import ActionHeaderAllergies from "./example/ActionHeaderAllergies"
+
+<Badge />
+
+# Accessibility Guide for Terra Action Header
+
+
+## Why is this important?
+
+> Terra Action Header is an essential component used to identify a section of content and provide controls to manage the respective content. If Terra Action Header is not made accessible, it can prevent users from understanding content and necessary context to interact with the page.  
+>
+>Action Headers provide a visual and programmatic presentation of content structure and convey a meaningful purpose to the section. Action headers also commonly use icon-only buttons that must visually communicate an action. In addition, the buttons must be intuitive and have a programmatic association to other words on the page to ensure a unique and descriptive accessible control name to allow speech input device users can use them. Screen reader users also rely on the added programmatic context to understand what precisely a control may do. 
```

Comment:
```suggestion
> Action Headers provide a visual and programmatic presentation of content structure and convey a meaningful purpose to the section. Icon-only buttons are commonly used to visually communicate an action. Therefore, the buttons must be intuitive and have a programmatic association with other words on the page to ensure a unique and descriptive accessible control name. This will allow speech input devices to utilize them. Screen reader users will also rely on the added programmatic context to understand what precisely a control may do. 
```